sgxs-tools/src/sgx_detect/tests/mod.rs

```diff
@@ -336,14 +352,28 @@ impl Print for EpcSize {
     }
 
     fn print(&self, level: usize) {
+        fn epc_size_unit(total_size: u64) -> (f64, String) {
+            let mut epc_size = total_size as f64 / 1024.0 / 1024.0;
+            let mut epc_unit = "MiB".to_owned();
+            if epc_size >= 1024.0 {
+                epc_size /= 1024.0;
+                epc_unit = "GiB".to_owned();
+            }
+            (epc_size, epc_unit)
+        }
+
         if let Some(epc) = &self.epc {
-            println!(
-                "{:width$}{}: {:.1}MiB",
-                "",
-                self.name(),
-                epc.total_size as f64 / (1048576.),
-                width = level * 2
-            );
+            print!("{:width$}{}:", "", self.name(), width = level * 2);
+            if epc.total_size_cip > 0 {
+                let (epc_size, epc_unit) = epc_size_unit(epc.total_size_cip);
+                print!(" {:.1}{} (Confidentiality and Integrity Protected)", epc_size, epc_unit);
```

Comment:
The terms Intel use are unclear, I don't think we should reuse them in our output. I'm still clarifying with Intel what future types will look like. Let's use something like this:

* regular SGX:
```
EPC size: 50.0MiB
```

* Ice lake server:
```
EPC size: 5.0GiB (no integrity protection)
```